types/@types/AccountIngress.d.ts

```diff
@@ -0,0 +1,328 @@
+// File copied from https://github.com/ConsenSys/permissioning-smart-contracts, so we can use AccountIngress type
```

Comment:
I'm pretty sure we don't need these copied over files. `web3.eth.Contract` type is plenty enough. If we wanted to have typed contracts in JS I would certainly start with our bridge contract, which has ton of functions, unless this one, where we need to call a single one. Can you try to remove them?

Reply:
Hmm, yes of course I can try to remove that. I wanted to add as much typing as possible but we can decrease that.

Reply:
done in the newest commit